make_functions.sh

```diff
@@ -149,6 +150,23 @@ build_push_operator_image() {
     fi
 }
 
+seed_repository() {
+  set -x
+  docker pull "docker.io/jujusolutions/juju-db:${JUJU_DB_VERSION}"
+	docker tag "docker.io/jujusolutions/juju-db:${JUJU_DB_VERSION}" "${DOCKER_USERNAME}/juju-db:${JUJU_DB_VERSION}"
+	docker push "${DOCKER_USERNAME}/juju-db:${JUJU_DB_VERSION}"
+
+  # copy all the lts that are available
+  for (( i = 18; ; i += 2 )); do
```

Comment:
Wowza

Reply:
Look, it does what is says on the can. 🥫